assessments/0008-backstage.md

```diff
@@ -0,0 +1,403 @@
+
+The website **typography** is easy to read.
+
+Examples:
+
+* https://helm.sh/
+
+
+#### Case studies/social proof
+
+**Case studies** and **testimonials** are not prominent and are not readily findable from the website. There is no **logo wall** of participating organizations.
+
+The project has an **active blog**. **Community discussions** are available on the website.
+
+Examples:
+
+* https://www.fluentd.org/testimonials (user testimonials)
+* https://goharbor.io/ (logo wall)
+* https://blog.rook.io/ (blog)
+
+
+#### SEO, Analytics and site-local search
+
+**Analytics**
+
+  - **Analytics** is enabled for the production server using **GA4**.
+  - Analytics is **disabled for all other deploys**? 
+  - **Page-not-found (404) reports** are easily generated from the site.
+
+**Site indexing**
+
+ **Indexing** is supported for the **production server**. Indexing is **disabled for previews non-default builds** automatically with `plugin-sitemap`.
+
+**Search**
+
+Local intra-site **search** is available from the website.
+
+**Custodians**
+
+The current **custodian(s)** of the following accounts are **not** clearly documented: analytics, Google Search Console, site-search (such as Google CSE or Algolia).
+
+
+#### Maintenance planning
+
+The **website tooling** (Docusaurus static site build) is well supported.
+
+**Cultivation of website maintainers** from within the community is inadequate.
+
+I tested the instructions for using `yarn` to build the website. The **site build time** was under 30 seconds for a local build on a Mac M1. Maintainers have sufficient **permissions** to download and build the doc. Checking in the doc requires a PR and approval from a project maintainer.
+
+Examples:
+
+* http://kubernetes.io
+
+#### Usability, accessibility and devices
+
+Backstage.io is partially conformant with [WCAG 2.1 level AA](https://www.w3.org/WAI/WCAG21/Understanding/) with respect to **accessibility**.
+
+[accessibility]: https://developer.mozilla.org/en-US/docs/Web/Accessibility
+
+
+### Recommendations
+
+#### Single-source repo
+
+The documentation is adequately isolated from the code by virtue of being in its own directories; however, its location and build instructions are obscure. Write explicit instructions for contributing to documentation. Emphasize the importance of keeping the documentation directories separate. Make these instructions prominent in the contributor guidelines.
+
+
+#### Minimum website requirements for maturity level
+
+To meet the maturity level standards for a graduated project, update the following aspects as described in [Project documentation](Project-documentation):
+
+- Identify the project and product stakeholder roles.
+- Analyze stakeholder needs.
+- Update and reorganize the documentation with respect to user orientation and task-based support of use cases.
+
+
+#### Case studies/social proof
+
+Implement a **logo wall** of participating organizations, with links to testimonials and/or case studies.
```

Comment:
I'm still looking into how best to ask for help to create case studies. Seems like a lot of case studies are driven from the user side.

Reply:
You could create issues to write case studies. Potentially an interesting change of pace for tech writers who want to contribute to a project.